<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="detekcja_zajezdnia_checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zajezdnia</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detekcja_perony_checkBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perony</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detekcja_przeszkody_checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Przeszkody</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detekcja_reka_checkBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ruch ręką</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WYKRYWANE OBIEKTY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DETEKCJA NA NAGRANIU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_nagranie_start">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DETEKCJA NA STRUMIENIU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stream_start">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_kalibruj">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KALIBRUJ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KALIBRACJA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detekcja_twarz_checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Twarz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detekcja_ruch_checkBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ruch pociągu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detekcja_czerwony_pociag_checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Czerwony pociąg</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
